browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -42,6 +44,27 @@ bool SpeedreaderTabHelper::IsSpeedreaderEnabled() const {
   return SpeedreaderServiceFactory::GetForProfile(profile)->IsEnabled();
 }
 
+bool SpeedreaderTabHelper::GetSiteSpeedreadable() {
```

Comment:
having `IsSpeedreaderEnabled` and `GetSiteSpeedreadable` seems a bit confusing. Why do we need two separate methods? If we do need both, I think this should be `IsEnabledForSite`